### ISSUE TITLE
Modifying value before validation

### ISSUE THREAD

**Issue by meredevelopment** [vuelidate/vuelidate#555]
I have a currency field which must contain a value between 1 and 30000. However UK users will likely enter monetary values with commas in i.e. "25,000" - so before doing any numeric validation I need to strip the `,` chars. 

I'm doing this: Is there a better way?

``` 
const money = helpers.regex('money', /^[\d\,]*$/)

validations() {
    return {
        contract_value: {
            required,
            maxValue: function(value) {
                return value.replace(/[^\d]/g, "") <= 30000;
            },
            minValue: function(value) {
                return value.replace(/[^\d]/g, "") >= 1;
            },
            money,
        },
    }
},
```

Seems like I should be able to improve this by not defining the regex twice, but I don't know how/where to do that.

Also I guess I don't need to be using `maxValue` and `minValue` as they already exist and are just being redefined here?

Basically ^^ code smell. Please help me improve it. Thanks!

**Comment by dobromir-hristov** [comment]
You should create a component that formats user input, yet spits plain integers.

**Comment by meredevelopment** [comment]
Ok, I don't understand that very well. If someone get's any time to throw some sample code up here, or expand that explanation, I would appreciate it a lot.

Thanks.
